infra/build/build_status/update_build_status.py

```diff
@@ -29,6 +29,7 @@
 import build_and_run_coverage
 import build_lib
 import build_project
+import fuzz_introspector_page_gen
```

Comment:
put this after 35 please (those imports aren't really compliant with our style as is, but this is more alphabetical).